code/run_capsule.py

```diff
@@ -1,7 +1,132 @@
-""" top level run script """
+import json
+import logging
+from datetime import datetime
 
-def run():
-    """ basic run function """
-    pass
+import data_contract
+import pandas as pd
+import pynwb
+import utils
+from aind_behavior_core_analysis._core import DataStream
+from aind_behavior_core_analysis.utils import load_branch
+from dateutil import parser
+from hdmf_zarr import NWBZarrIO
+from ndx_events import EventsTable, MeaningsTable, NdxEventsNWBFile
 
-if __name__ == "__main__": run()
\ No newline at end of file
+logger = logging.getLogger(__name__)
+
+if __name__ == "__main__":
+    logging.basicConfig(
+        level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+    )
+
+    primary_data_path = tuple(utils.DATA_PATH.glob("*"))
```

Comment:
I would not use a hardcoded path here. Users should have the option to add in their own input and output directories

Reply:
I would always code with that in mind. I think `argparse` or `BaseSettings` will work